tools/ansible/roles/dockerfile/templates/Dockerfile.j2

```diff
@@ -199,6 +199,10 @@ ADD tools/ansible/roles/dockerfile/files/rsyslog.conf /var/lib/awx/rsyslog/rsysl
 ADD tools/ansible/roles/dockerfile/files/wait-for-migrations /usr/local/bin/wait-for-migrations
 ADD tools/ansible/roles/dockerfile/files/stop-supervisor /usr/local/bin/stop-supervisor
 
+# Create default awx rsyslog sidecar
```

Comment:
this does not create the side car 

a "sidecar" is essentially another container that's deployed in the same pod as your "main" container

https://docs.docker.com/engine/reference/builder/

this is the dockerfile documentation and ADD adds a file to your docker container image 

this statement below means "add the supervisor_rsyslog.conf.j2 template to the container image and renamed it to /var/run/supervisor/supervisor.rsyslog.sock"

lets talk about what you are trying to do with this statement

Reply:
for additional context

dockerfile builds docker container **image** 
when you create a container via kubernetes deployment it uses the image as basis to start the container

Reply:
https://docs.docker.com/engine/reference/builder/#add

Reply:
Thank you so much for this context! I reviewed the documentation and it looks like I need to tell it where it will render:

Reply:
Actually, upon further investigation, I don't think I need this at all.

Reply:
Thank you so much for your reply! I think it is in order now.